convert2rhel/systeminfo.py

```diff
@@ -87,6 +87,7 @@ def __init__(self):
         self.default_rhsm_repoids = None
         # List of repositories enabled through subscription-manager
         self.submgr_enabled_repos = []
+        # Gets the release version of the Linux based system, see the docstring in the “get_releasever()”
```

Comment:
```suggestion
        # Holds the release version of the Linux based system, see the docstring in the “get_releasever()”
```

I would say holds since it's a variable.  I'd use `get` for a description of a function.  (variables are generally nouns and functions are verbs)